lib/identity/Identity.js

```diff
@@ -71,13 +72,29 @@ class Identity {
     return this.publicKeys.find((k) => k.getId() === keyId);
   }
 
+  /**
+   * Get plain object representation
+   *
+   * @return {Object}
+   */
+  toObject() {
+    return {
+      protocolVersion: this.getProtocolVersion(),
+      id: this.getId().toBuffer(),
+      publicKeys: this.getPublicKeys()
+        .map((publicKey) => publicKey.toObject()),
+      balance: this.getBalance(),
+      revision: this.getRevision(),
+    };
+  }
+
   /**
    * @return {RawIdentity}
    */
```

Comment:
```suggestion
  /**
   * Get Identity JSON representation
   * 
   * @return {JsonIdentity}
   */
```

I guess we need to define JSON representation types for all structures.